great_tables/_tbl_data.py

```diff
@@ -197,11 +197,13 @@ def _(data, row: int, column: str, value: Any) -> None:
     # if this is violated, get_loc will return a mask
     col_indx = data.columns.get_loc(column)
     data.iloc[row, col_indx] = value
+    return data
```

Comment:
Since `_set_cell()` mutates data, it seems useful to continue not returning `data` (sometimes called [Command query separation](https://en.wikipedia.org/wiki/Command%E2%80%93query_separation)).